UPGRADE-14.0.md

```diff
@@ -195,8 +195,25 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -   fix Comparison for not logged in users ([#2905](https://github.com/shopsys/shopsys/pull/2905))
     -   unified code for Comparison and Wishlist
     -   refactored Zustand store to use only one store (User Store) for all cartUuid, wishlistUuid and comparisonUuid
--   added skeletons for Wishlist, Comparison and My orders, Order detail and Product Variants page ([#2906](https://github.com/shopsys/shopsys/pull/2906))
-    -   see #project-base-diff to update your project
+-   added skeletons for Wishlist, Comparison and My orders, Order detail and Product Main Variant page ([#2906](https://github.com/shopsys/shopsys/pull/2906))
 -   fix Add to cart/wishlist/comparison feature when using multiple tabs ([#2906](https://github.com/shopsys/shopsys/pull/2906))
-    -   As fresh unlogged user without `cartUuid` I open multiple tabs, then in each tab I add product to cart, I am experiencing that each tab contains different products in cart, even after refresh. Same applies to `Wishlist` and `Comparison`. This was solved by adding `BroadcastChannel` middleware to Zustand store, so now it uses always only first returned `uuid`.
-    -   see #project-base-diff to update your project
+
+    -   Before this change if I open multiple tabs as fresh unlogged user without `cartUuid`, then in each tab I add product to cart, I am experiencing that each tab contains different products in cart, even after refresh. Same applies to `Wishlist` and `Comparison`. This is solved now by adding `BroadcastChannel` middleware to Zustand store, so now it uses always only first returned `uuid`.
+    -   removed redirect to homepage after logout, this was not good approach
+    -   `removeItemFromCartAction ` in `useRemoveProductFromCart` is now `void` type (doesn't return `CartFragmentApi`) and in case of error is returned from API it refect cart so user gets latest cart updates
+    -   Zustand package is upgraded to latest version
+    -   now we hydrate Zustand store manually during first load, which means that now we don't have do overcomes for matching server rendered UI with client like
```

Comment:
Phrase _which means that now we don't have do overcomes for matching server rendered UI with client_ seems incorrect. Try changing it to something more appropriate, such as:

_We are now manually hydrating the Zustand store during the initial page load. This means that we no longer need to resolve mismatches between the UI rendered by the server and the UI managed on the client side_.

Reply:
Done.